x/ecocredit/migrations/v3/types.pb.go

```diff
@@ -466,11 +466,11 @@ func (m *CreditTypeSeq) GetSeqNumber() uint64 {
 }
 
 func init() {
-	proto.RegisterType((*ClassInfo)(nil), "regen.ecocredit.v1alpha1.ClassInfo")
-	proto.RegisterType((*BatchInfo)(nil), "regen.ecocredit.v1alpha1.BatchInfo")
-	proto.RegisterType((*Params)(nil), "regen.ecocredit.v1alpha1.Params")
-	proto.RegisterType((*CreditType)(nil), "regen.ecocredit.v1alpha1.CreditType")
-	proto.RegisterType((*CreditTypeSeq)(nil), "regen.ecocredit.v1alpha1.CreditTypeSeq")
+	//proto.RegisterType((*ClassInfo)(nil), "regen.ecocredit.v1alpha1.ClassInfo")
```

Comment:
Why these lines commented?

Reply:
We register the types here: https://github.com/regen-network/regen-ledger/pull/1360#discussion_r949333351